.github/workflows/trivy-scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trivy Scan
+
+on:
+  push:
+    # Run on main and supported release branches
+    branches:
+      - main
+      - release-1.23
+      - release-1.22
+      - release-1.21
+  # Run weekly
+  schedule:
+    - cron: '0 12 * * 1'
```

Comment:
Unfortunate that we can't schedule runs against the release branches -- docs say "Scheduled workflows run on the latest commit on the default or base branch".

Reply:
yeah we could maybe try something that has a matrix of branch names to do the checkout/scan/upload steps and only have it run on a schedule and manually rather than on push?

Reply:
or have multiple workflows which is a little unfortunate

Reply:
Yeah that sounds reasonable, I'm fine getting this in as an initial step though since we can run manually.